fedimint-client/src/module/init/recovery.rs

```diff
@@ -238,7 +238,7 @@ where
             epoch_range: ops::Range<u64>,
         ) -> impl futures::Stream<Item = (u64, Vec<AcceptedItem>)> + 'a {
             // How many request for blocks to run in parallel (streaming).
-            const PARALLISM_LEVEL: usize = 8;
+            const PARALLISM_LEVEL: usize = 64;
```

Comment:
Can confirm, I went for 32 in `fedimint-observer`, but 64 is probably better for mobile with higher latencies.